auth/authenticators/iam-authenticator.ts

```diff
@@ -14,17 +14,39 @@
  * limitations under the License.
  */
 
-import { OutgoingHttpHeaders } from 'http';
 import { IamTokenManager } from '../token-managers';
 import { validateInput } from '../utils';
-import { BaseOptions, TokenRequestBasedAuthenticator } from './token-request-based-authenticator';
+import { BaseOptions, TokenRequestBasedAuthenticator }
+  from './token-request-based-authenticator';
 
+/**
+ * Configuration values for [[IamAuthenticator]] behavior. Requires an `apikey`
```

Comment:
Remove the required part of the description. Remove behavior word. Interface is only for typescript, and it is not needed here.